free-1.md

```diff
@@ -4,40 +4,130 @@
+Furthermore, the game is balanced around a theoretical limit of human control.
+For instance, in Starcraft II it is common knowledge that 66 Terran Siege Tanks
+will defeat 800 Zerglings if charged blindly into the enemy, as shown in
+[this video](https://www.youtube.com/watch?v=SS6totiGZik) -- please ignore the
+voices and low frame rate. That being said, modders, using bots superhuman CPU
+reaction times, have shown that it is possible for 100 Zerglings to defeat
+20 Terran Siege Tanks if individually controlled with enough persision, as
+seen in [this video](https://www.youtube.com/watch?v=IKVFZ28ybQs). Although
+tools to script commands do exist, they are usually done by programmers after
+the fact for fun and are not core functions of the game. In fact, most games
+would consider any kind of computer aided control tool to be a form of cheating
+since it provides users with an unfair advantage. This advantage is made very
+clear in
+[this comparison video](https://www.youtube.com/watch?v=DXUOWXidcY0)
+between a top tier professional player and a bot designed to do the same task.
+
+Furthermore, although compititions exist to design AIs to play the games,
+one such is UC Berkeley's "Berkeley Overmind" (as seen in 
+[this video](https://www.youtube.com/watch?v=fWtpTu9gB20)) which plays
+Starcraft with a superhuman average actions per minute (APM) of 3397 and a
+peak performance over 7000 APM, no game to my knowledge incorperates coding
+natively for fine grained unit control.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Since this language aims to allow gamers to control units beyond the bounds of
+a typical human reaction time, there must be some coding interface to allow
+users to express their desired behaviors in game. Thus, by allowing users
+to express a controls in code form, they can be sped up to superhuman speeds.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Although this concept makes for an interesting game idea, the real reason I
+am excited about designing this game is because of its educational value. I
+started playing Starcraft back in 4th grade and its modding tools 
+were my original introduction to programming. Although this game should be
+ideally enjoyable to all programmers, I imagine if the levels are designed
+well with a good progression in difficult, this game could be used to introduce
+completely novice programmers to coding. I've always been amazed with how
+even the most monotonous tasks can be made "fun" in the context of a video
+game by occasionally adding some flashing lights and a "level-up" screen.
+Although I personally think programming is fun to begin with, I think by
+wrapping core coding skills within a enteraining game will help coding
+become more accessable to gamers.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Hardcore gamers teach novice programming.
```

Comment:
Is this true? Or is it the game itself that teaches the programming?

This is an exciting idea so far!